modules/apps/commerce/commerce-notification-test/src/testIntegration/java/com/liferay/commerce/notification/test/CommerceNotificationTest.java

```diff
@@ -182,23 +181,6 @@ public void testEmailAddressRecipient() throws Exception {
 		Assert.assertEquals(1, commerceNotificationQueueEntriesCount);
 	}
 
-	@Test(expected = NoSuchUserException.class)
```

Comment:
@Commando18 why do we remove test, instead of adding new one?

Reply:
ok, talked with @Commando18 and this test was removed because with the latest changes this use case does not exist anymore